UnityProject/Assets/Scripts/UI/Systems/Lobby/SubCustomisation/BodyPartCustomisations/BodyPartDropDownReplaceOrgan.cs

```diff
@@ -122,6 +122,12 @@ public void DropdownScrollLeft()
 		}
 	}
 
+	public void RandomizeDropdownValue()
```

Comment:
you're supposed to implement randomise in BodyPartCustomisationBase  and then then override it from the classes that inherit from it

Reply:
Since dropdowns aren't always present on all classes that inherit from Base I didn't see the point in making a virtual function on the base script for that

Reply:
well it makes it so you don't have to get component BodyPartDropDownReplaceOrgan you can just get component BodyPartCustomisationBase